src/e2e-app/expansion/expansion-e2e.html

```diff
@@ -0,0 +1,21 @@
+<mat-accordion>
```

Comment:
Instead of adding a separate e2e page here, can you test an example material-examples?